### PR TITLE
Feature/31 support conf files

### PR DESCRIPTION
**Pull request by grigobog**

### CODE REVIEW

File: bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java [hunk 1]
```diff
@@ -94,6 +94,22 @@ public void mapAndWriteTestNew() {
...
     }
 
+    @Test
+    public void testReadActorMappingFile() {
+        ConverterMappingService converterMappingService = new ConverterMappingService(
+                "prefix-bpmn",
+                "prefix-organization-structure",
+                "prefix-pre-stamp"
+        );
+        ActorMappings actorsFromXml = converterMappingService.readActorMappingFile(new File("src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/data/actor_mapping_1.xml"));
```
Comment (blcham) [inline bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java:104]: IMHO, we should rather use resources that path to file so it works even if executed from parent project or elsewhere.  Is it complicated to use like that?

File: bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java [hunk 2]
```diff
@@ -94,6 +94,22 @@ public void mapAndWriteTestNew() {
 //        );
     }
 
+    @Test
```
Comment (blcham) [inline bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java:97]: BTW, what is cz.cvut.kbss.bpmn2stamp.**wip**.bpmn2java ? What is wip ? Work in progress ?

**Review by blcham — APPROVED** [review]
LGTM, consider resolving my comments, can be merged afterwards.

File: bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java (hunk 1 above)
Comment (blcham) [reply bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java:104]: data --> should be `/src/test/resources`

**Comment by blcham** [comment]
It was tested, it works correctly. After resolving comments, it can be merged.

File: bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java (hunk 1 above)
Comment (grigobog) [reply bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java]: fixed

File: bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java (hunk 2 above)
Comment (grigobog) [reply bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java]: removed in further commits

**Pull request merged**